datadog_checks_dev/datadog_checks/dev/tooling/commands/release/testable.py

```diff
@@ -291,13 +293,9 @@ def testable(ctx: click.Context, base_ref: str, target_ref: str, milestone: str,
         pr_author = pr_data.get('user', {}).get('login', '')
         pr_body = pr_data.get('body', '')
 
-        jira_config = user_config['jira']
-        if not (jira_config['user'] and jira_config['token']):
-            abort('Error: You are not authenticated for Jira. Please set your jira ddev config')
```

Comment:
Would this be helpful to keep for Trello?

Reply:
Sure, I'll add it back!